### DISCUSSION TITLE
How to get the insights of Atoti?

### DISCUSSION THREAD

**Discussion by tht-activeviam** [atoti/atoti#598] [Q&A]
How can we get the insights of running application?
Example:
- Number of logged in user
- Running queries at the moment
Is there a way to expose those metrics through REST?

**Comment by tibdex** [comment]
Hi Tin, this is not supported at the moment. Can you explain why you need these metrics? Are there other metrics you would be interested in?

**Reply by tht-activeviam** [reply]
Hi Thibault,
This is one of the item in the checklist by production support.
